fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Lao.java

```diff
@@ -80,13 +79,8 @@ public Lao(String id) {
   }
 
   public Lao(String name, PublicKey organizer, long creation) {
+    // This will throw an exception if name is null or empty
     this(generateLaoId(organizer, creation, name));
-    if (name == null) {
-      throw new IllegalArgumentException("The name of the Lao is null");
-    }
-    if (name.isEmpty()) {
-      throw new IllegalArgumentException("The name of the Lao is empty");
-    }
```

Comment:
This was unreachable. Indeed an empty or null name would trigger a `IllegalArgumentException` from `generateLaoId`